### PR TITLE
ENG-2199-multiple-configs

### PR DESCRIPTION
**Pull request by tsiq-kylel**
Also updated readme and a few other small changes:
* zip output param renamed to "zipDest"
* Default dockerrun template fixed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ This plugin simplifies the build and deploy steps of applications using Docker a
 
 ### Set up
 
-Currently, we only support pushing to AWS ECR as a docker registry. This requires AWS credentials. 
+Currently, we only support pushing to AWS ECR as a docker registry. This requires AWS credentials.
```
Comment (tsiq-hanxie) [inline README.md]: Could you delete / change this line since we're no longer supporting just ECR.

File: pom.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
   <groupId>com.twosigmaiq</groupId>
   <artifactId>magic-beanstalk-maven-plugin</artifactId>
-  <version>2.0.0</version>
+  <version>1.0.6</version>
```
Comment (tsiq-hanxie) [inline pom.xml]: Why did we bump the release version to 2.0?

Comment (tsiq-kylel) [reply pom.xml]: No idea, it was bumped in the auth merge

File: src/main/java/com/twosigmaiq/magic/beanstalk/util/ZipUtil.java [hunk 3]
```diff
@@ -24,6 +24,10 @@
    * @param output Zip filename as a string.
    */
   public static void writeZip(List<String> files, String output) throws IOException {
+    // Clear the zipFiles folder.
```
Comment (tsiq-hanxie) [inline src/main/java/com/twosigmaiq/magic/beanstalk/util/ZipUtil.java]: The indenting's a bit weird in this file. Can you install the google java style guide into your intellij? https://github.com/HPI-Information-Systems/Metanome/wiki/Installing-the-google-styleguide-settings-in-intellij-and-eclipse.

Comment (tsiq-hanxie) [reply src/main/java/com/twosigmaiq/magic/beanstalk/util/ZipUtil.java]: nvm it looked different in github but looks fine in intellij...

File: pom.xml (hunk 2 above)
Comment (tsiq-hanxie) [reply pom.xml]: My mistake thanks for that catch. My memory must be going bad in my old age 👴

**Pull request merged**